cdb2jdbc/src/main/java/com/bloomberg/comdb2/jdbc/Comdb2Handle.java

```diff
@@ -1726,7 +1730,7 @@ private boolean reopen(boolean refresh_dbinfo_if_failed) {
          */
 
         // last time we were at dbHostIdx, this time start from (dbHostIdx + 1)
```

Comment:
Fix the issue where the JDBC driver seemingly would not connect.  dbHostIdx was 0, and myDbHosts.size() was 1, which means that prior to this change, both of the next two for loops (which attempt to re-establish a connection with a different replicant) were skipped.